### ISSUE TITLE
Acacia trees

### ISSUE THREAD

**Issue by mazes-80** [joe7575/lumberjack#20]
Acacia trees sometimes grow with just one stem and make branches immediately.
Lumberjack mod is not able to cut "whole" acacia tree in this case.

**Comment by mazes-80** [comment]
Schematic from side (A:air, T:acacia tree, D: soil)
`TAAAT            TAAAT`
`ATATA            ATATA`
`AATAA -> chop -> AAAAA`
`DDDDD            DDDDD`

**Comment by joe7575** [comment]
> Acacia trees sometimes grow with just one stem and make branches immediately.

Standard acacia in a MTG world?

**Comment by mazes-80** [comment]
Le Sun, 19 Mar 2023 10:05:38 -0700,
Joachim Stolberg ***@***.***> a écrit :
 
> Standard acacia in a MTG world?
> 
The world I play in with lumberjack mod is a MTG world.
There are no more trees than default ones in.

**Comment by joe7575** [comment]
I have never seen that. 
But you could try if the following code change solves the problem without any other side effects.
Change https://github.com/joe7575/lumberjack/blob/master/init.lua#L424 
from:
```
lumberjack.register_tree("default:acacia_tree", "default:acacia_sapling", 2, 3)
```
to
```
lumberjack.register_tree("default:acacia_tree", "default:acacia_sapling", 2, 1)
```

**Comment by mazes-80** [comment]
Le Mon, 20 Mar 2023 11:39:39 -0700,
Joachim Stolberg ***@***.***> a écrit :

I tested only a bit with:
2,1) breaks all
2,2) radius is too small some tree part remains
3,2) seems to work, but testing was not intensive

So I will keep
```
lumberjack.register_tree("default:acacia_tree", "default:acacia_sapling", 3, 2)
```

An comment back after a few time testing

**Comment by mazes-80** [comment]
My schema about acacia tree was inaccurate
Schematic from side (A:air, T:acacia tree, D: soil)
`TAAAT            TAAAT`
`ATTTA            ATTTA`
`AATAA -> chop -> AAAAA`
`DDDDD            DDDDD`

`lumberjack.register_tree("default:acacia_tree", "default:acacia_sapling", 3, 2)`

Solves the case and no weird things happened until now. My acacia tree dedicated node breaker was never stuck since this change

**Comment by mazes-80** [comment]
Nice

**Comment by mazes-80** [comment]
Le Mon, 01 May 2023 03:10:38 -0700,
Joachim Stolberg ***@***.***> a écrit :

> Closed #20 as completed.
> 

Oh, I just see you use 2,2

Seems I tested and had some remaining trunks for high trees.
Not sure anyway as testing dates a bit
I'll double check this.
